proto/pki/tx.proto

```diff
@@ -22,6 +22,8 @@ service Msg {
 message MsgProposeAddX509RootCert {
   string signer = 1 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.moretags) = "validate:\"required\""];
   string cert = 2 [(gogoproto.moretags) = "validate:\"required,max=10485760\""];
+  string info = 3 [(gogoproto.moretags) = "validate:\"max=4096\""];
```

Comment:
We should extend `x/pki/types/message_xxx_test` files to add a test for the lower and upper bound for the `info` field (that "" info is accepted and that len(info) > 4096 is not accepted).

Reply:
Added the upper and lower bound checks